executor/runcexecutor/monitor_stats.go

```diff
@@ -52,6 +52,9 @@ func (w *runcExecutor) monitorContainerStats(ctx context.Context, id string, sam
 
 		stats, err := w.runc.Stats(ctx, id)
 		if err != nil {
+			if errors.Is(err, context.Canceled) {
+				return
+			}
```

Comment:
Just double-checking that this is also an intended change as part of this PR

Reply:
yes -- it is, it's unfortunate github doesn't update the description when a git rebase occurs, I changed to the commit message to this:

```
dont log context canceled errors
A context canceled error will occur once the runc container has exited,
there is not need to log these errors.

Additionally, remove a printf that was accidentally committed.
```

but forgot to copy/paste it under the description here.

^^ github product owners take note :smile: